src/components/SearchBar/SearchBar.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react'
+
+import Dropdown from "../Dropdown/Dropdown";
+import SearchForm from "../SearchForm/SearchForm";
+import styles from "./SearchBar.module.scss";
+
+const SearchBar = props => {
+    return (
+        <div className={styles.SearchBar}>
+            <SearchForm 
+                theme={props.theme}
+                searchTxt={props.searchTxt}
+                searchHandler={props.searchHandler}
+            />
+            <Dropdown 
+                list={props.regionList}
+                theme={props.theme}
+                selectedItem={props.region}
+                onSelect={props.setRegion}
+                buttonTxt={props.buttonTxt}
```

Comment:
@ App.js there is no need to define a constant for a single label and then pass it down two components! Just do: `buttonText="Filter by Region"` right here and remove const in the App.